tests/e2e/utils.js

```diff
@@ -0,0 +1,23 @@
+import { browser, $ } from '@wdio/globals';
+
+let extensionId = '';
+export async function getExtensionId() {
+  if (!extensionId) {
+    await browser.url('chrome://extensions');
```

Comment:
this will influence the context of the test. 

maybe instead of checking in the runtime, we can include a `key` in the manifest to have a fixed extension id? 
like it is done here https://github.com/ghostery/webextension-event-recorder/blob/ae9005bc60d4385312730d8ea067f53230144fbb/extension/manifest.json#L6
(key will be there in debug builds only)

Reply:
The tests should run on the package closest to the one we publish. This is the recommended way to get the extension id from the WDIO docs. What do you mean by that it will influence the context? How can opening the settings page impact tests?

Reply:
This function will load `extensions` page and keep it open (maybe you should just close the tab after the id is acquired). And it will happen first time the extensionid is accessed, which may not be expected behavior when you filter tests.


Including `key` in the manifest should be safe as  `key` is does not influence the extension behavior.


Also, this exact method wont work on Firefox.

Reply:
The WDIO uses the same tab to navigate. You can try out the `npm run build && npm run wdio:debug` to see how it works.